celery-stubs/concurrency/base.pyi

```diff
@@ -1 +1,70 @@
-class TaskPool: ...
```

Comment:
TaskPool doesn't exist in latest Celery versions in this file, changed all usages to `BasePool`.